docs/src/modules/components/MaterialFreeTemplatesCollection.js

```diff
@@ -9,66 +9,62 @@ import Typography from '@mui/material/Typography';
 import Visibility from '@mui/icons-material/Visibility';
 import CodeRoundedIcon from '@mui/icons-material/CodeRounded';
 import { useTranslate } from '@mui/docs/i18n';
-import { useTheme } from '@mui/material/styles';
 
 const sourcePrefix = `${process.env.SOURCE_CODE_REPO}/tree/v${process.env.LIB_VERSION}`;
 
-function layouts(translatation, theme) {
-  const imageSuffix = theme.palette.mode === 'dark' ? '-dark' : '-light';
+function layouts(translatation) {
   return [
     {
       title: translatation('dashboardTitle'),
       description: translatation('dashboardDescr'),
-      src: `/static/images/templates/dashboard${imageSuffix}.png`,
       href: '/material-ui/getting-started/templates/dashboard/',
       source: `${sourcePrefix}/docs/data/material/getting-started/templates/dashboard`,
+      hasDarkMode: true,
```

Comment:
I think that'd be overkill for now. It can be a separate PR but need help from @zanivan to find a good DX to show between custom/default with light/dark.

Reply:
I plan to use the default image in a blog post (separate PR), that's the reason I added it.